File: src/ProjectR.Scripting/content/scripts/spells/lua/ForbiddenSpell98.lua
function GetTargetType()
    return Single
end

function GetName()
return "Forbidden Spell #98"
end

function GetDescription()
  return "Focuses all your arcane power into\nannihilating your target.\nIgnores most of target's MR."
end

function IsSupportSpell()
    return false
end

function GetMPCost()
    return 2
end

function GetDelay()
  return .4
end

function GetSpellType()
  return Pure
end

function GetMasteries()
  return { ARC }
end

function SpellEffect()
  local damage = 10 * aMD * (aARC/100) - .3 * dMR
  d:TakeDamage(damage)
end

File: src/ProjectR.Scripting/content/scripts/spells/ForbiddenSpell98.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting.Spells
{
    public class ForbiddenSpell98 : SpellScriptBase
    {
        public override string Name { get { return "Forbidden Spell #98"; } }
        public override string Description { get { return "Focuses all your arcane power into\nannihilating your target.\nIgnores most of target's MR."; } }

        public override TargetType TargetType { get { return TargetType.Single; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { EleMastery.ARC, }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return false; } }

        public override double MPCost { get { return 200; } }
        public override double Delay { get { return .4; } }

        public override void SpellEffect(ICharacter caster, ICharacter target)
        {
            var damage = 10 * cMD * (cARC / 100) - .3 * tMR;
            DealDamage(damage);
        }
    }
}